### ISSUE TITLE
Cannot install version 1.2.9 (fails at installing deepvariant trying to link non-existent google cloud account)

### ISSUE THREAD

**Issue by tpham2654** [bcbio/bcbio-nextgen#3624]
I cannot install the latest version of BCBio at all.  I have attempted to install this "from scratch" as suggested by deleting the entire old BCBio installation (version 1.2.8) first .  It consistently fails when trying to install deepvariant with a message that 
```
Your credentials are invalid. Please run
$ gcloud auth login

return code: 1

()
```
The gcloud program was installed at one point on the system, but I removed it since it is no longer needed and I am still getting the error even after it.  Obviously running "gcloud auth login" does nothing since it is removed.  How to I get past this error, whether by successfully installing deepvariant or skipping over it?  This is a desktop computer server, not a cluster or cloud system.  We have no intention of linking google cloud to it at all.

**Version info**
- bcbio version (`bcbio_nextgen.py --version`): 1.2.9
- OS name and version (`lsb_release -ds`): CentOS Linux release 7.9.2009 (Core)

**To Reproduce**
Exact bcbio command you have used:
```
python3 bcbio_nextgen_install.py ./bcbio --tooldir=./bcbio/tools --nodata --mamba
```
Even if I do not include the "--mamba" flag I still get the same error.

**Comment by naumenko-sa** [comment]
Hi @tpham2654 !

There was a similar issue a while ago - could you please first try suggestions outlined there?
https://github.com/bcbio/bcbio-nextgen/issues/2927

Sergey

**Comment by tpham2654** [comment]
I am still getting the same error.  In fact, there is no "gcloud" executable or file in the ./bcbio/anaconda/envs/dv/bin folder either.

**Comment by naumenko-sa** [comment]
Have you updated your PATH after installing bcbio? 
https://bcbio-nextgen.readthedocs.io/en/latest/contents/intro.html#run-the-analysis-distributed-on-8-local-cores-with

You should be able to use `mamba install -c bioconda google-cloud-sdk -n dv`

**Comment by tpham2654** [comment]
As far as I can tell it did not successfully install.  However, the path already contains the BCBio directory which was set from a previous version.  I deleted the contents of that folder before trying to reinstall.

**Comment by naumenko-sa** [comment]
That means that dv environment has not been created.

Could you please 
1) provide the full installation log
and the output of the commands
2) `echo $PATH`
3) `echo $PYTHONPATH`
4) `which python`
5) `which -a conda`
6) `conda info -e` - list of environments
7) `which -a gsutil`
8) `ls ~/.gsutil`

Thanks!

**Comment by tpham2654** [comment]
Here are the outputs of the commands:
```
echo $PATH
/home/xxu/programs/vep/htslib:/home/xxu/programs/apache-maven-3.6.3/bin:/home/xxu/programs/vep/samtools/bin:/home/xxu/programs/bcbio/tools/bin:/home/xxu/programs/sratoolkit.2.9.6-1-centos_linux64/bin:/home/xxu/programs/baseSpace:/home/xxu/miniconda3/condabin:/home/xxu/programs/vep/htslib:/home/xxu/programs/apache-maven-3.6.3/bin:/home/xxu/programs/vep/samtools/bin:/home/xxu/programs/bcbio/tools/bin:/home/xxu/programs/sratoolkit.2.9.6-1-centos_linux64/bin:/home/xxu/programs/baseSpace:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/bin:/sbin:/opt/mssql/bin:/opt/mssql-tools/bin:/home/xxu/.local/bin:/home/xxu/bin:/opt/mssql/bin:/opt/mssql-tools/bin

echo $PYTHONPATH
(empty)

which python
/usr/bin/python

which -a conda
~/miniconda3/condabin/conda

conda info -e
# conda environments:
#
base                  *  /home/xxu/miniconda3
UCOP                     /home/xxu/miniconda3/envs/UCOP
mageck-vispr             /home/xxu/miniconda3/envs/mageck-vispr
pvactools                /home/xxu/miniconda3/envs/pvactools
                         /home/xxu/programs/bcbio/anaconda
                         /home/xxu/programs/bcbio/anaconda/envs/bwakit
                         /home/xxu/programs/bcbio/anaconda/envs/dv
                         /home/xxu/programs/bcbio/anaconda/envs/htslib1.10
                         /home/xxu/programs/bcbio/anaconda/envs/htslib1.11
                         /home/xxu/programs/bcbio/anaconda/envs/htslib1.12
                         /home/xxu/programs/bcbio/anaconda/envs/htslib1.12_py3.9
                         /home/xxu/programs/bcbio/anaconda/envs/htslib1.9
                         /home/xxu/programs/bcbio/anaconda/envs/java
                         /home/xxu/programs/bcbio/anaconda/envs/python2
                         /home/xxu/programs/bcbio/anaconda/envs/python3.6
                         /home/xxu/programs/bcbio/anaconda/envs/r35
                         /home/xxu/programs/bcbio/anaconda/envs/rbcbiornaseq
                         /home/xxu/programs/bcbio/anaconda/envs/samtools0

which -a gsutil
/usr/bin/which: no gsutil in (/home/xxu/programs/vep/htslib:/home/xxu/programs/apache-maven-3.6.3/bin:/home/xxu/programs/vep/samtools/bin:/home/xxu/programs/bcbio/tools/bin:/home/xxu/programs/sratoolkit.2.9.6-1-centos_linux64/bin:/home/xxu/programs/baseSpace:/home/xxu/miniconda3/condabin:/home/xxu/programs/vep/htslib:/home/xxu/programs/apache-maven-3.6.3/bin:/home/xxu/programs/vep/samtools/bin:/home/xxu/programs/bcbio/tools/bin:/home/xxu/programs/sratoolkit.2.9.6-1-centos_linux64/bin:/home/xxu/programs/baseSpace:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/bin:/sbin:/opt/mssql/bin:/opt/mssql-tools/bin:/home/xxu/.local/bin:/home/xxu/bin:/opt/mssql/bin:/opt/mssql-tools/bin)

ls ~/.gsutil
credstore2  credstore2.lock  metrics.log  tracker-files
```
Can you please tell me where the installation log is located?

**Comment by tpham2654** [comment]
I managed to resolve the problem.  Basically, someone had added their google account to gcloud auth sometime in the past and it persisted even after gcloud in the base environment was removed, and was even visible to the gcloud dependency BCBio attempted to install into the dv environment.  What I did was:

1. Activate the dv environment: ```conda activate ~/programs/bcbio/anaconda/envs/dv```
2. Install the google cloud SDK which includes the gcloud command: ```conda install -c conda-forge google-cloud-sdk```
3. List accounts to see if something had been linked: ```gcloud auth list```
In my case, there was.
4. Remove all accounts: ```gcloud auth revoke --all```
5. Exit the environment: ```conda deactivate```
6. Re-run the installer script.

When I did this, it went all the way to completion.
